packages/odyssey-react/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React, { FunctionComponent } from 'react'
+
+export type TagListProps = {
+  /**
+   * Text content to be rendered within the tag, it should describe an entity.
+   */
+  tags: Array<string>
+}
+
+/**
+ * Use Tags to help describe and differentiate an entity or object. Think of them as “adjectives” in your UI toolbox 
+ * that make navigating and parsing content easier.
+ * 
+ * @component
+ * @example <Tag tags={['Winged', 'Reusable', 'NASA']} />
+ */
+const Tag: FunctionComponent<TagListProps> = ({ tags }) => (
+  <ul className="ods-tag--list" data-testid="ods-tag">
```

Comment:
see my feedback [in the test PR](https://github.com/okta/odyssey/pull/953) about removing `data-testid` attributes in favor of more meaningful assertions that will have less flake and zero consequence on our bundle size!